src/create-select.tsx

```diff
@@ -203,6 +204,7 @@ const createSelect = (props: CreateSelectProps) => {
     });
 
     element.addEventListener("focusout", (event: FocusEvent) => {
+      hideInput();
```

Comment:
This will always hide on focus out (even when we were selecting an option). However, in the case of `multiple` we want the cursor to remain visible after selecting an option. 

So we need to do this after the check for related elements loop. In other words, we should do this in this function just before the `close()` call.